agent/src/main/java/com/microsoft/hydralab/agent/runner/scheduled/ScheduledDeviceControlTasks.java

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-package com.microsoft.hydralab.agent.scheduled;
+package com.microsoft.hydralab.agent.runner.scheduled;
```

Comment:
Why do you want to move this file to the runner folder? There are 2 schedule tasks in the class, and they are used to keep WebSocket connection and clear files on the PC. And the runner is used to run tests like Espresso and Appium. I don't think this file is a part of runner.